ixofoundation/ixo-webclient
# chore: update top nav for app ixo world (#167)

## Scope 
Update top nav for app.ixo.world 
https://nonaredteam.atlassian.net/secure/RapidBoard.jspa?rapidView=77&projectKey=IXO&modal=detail&selectedIssue=IXO-422

## Work done
Updated top nav for desktop and mobile view for app.ixo.world

## Steps to test
Verify that top nav matches designs
https://www.figma.com/file/ok50elV5abdYseQYKaSioG/ixo.world-website-revisions?node-id=3%3A0

Verify that links go to correct pages (note: all new pages don't exist yet)

– - Explorer (app.ixo.world) - Takes you to app.ixo.world
– - Build (developers.ixo.world) - Takes you to developers.ixo.world
– - Deliver (deliver.ixo.world) - Takes you to app.ixo.world/getixowallet/deliver
– - Invest (invest.ixo.world) - Takes you to app.ixo.world/getixowallet/invest
– -   Learn  (ixo-assistant FAQ) - Does not take you anywhere for now.
(I've made this go to UnderConstruction page for now)

## Screenshot
- Desktop
<img width="1103" alt="Screenshot 2020-05-18 at 16 41 42" src="https://user-images.githubusercontent.com/35291291/82226182-76410100-9926-11ea-8163-a282829433b5.png">

- Mobile
<img width="763" alt="Screenshot 2020-05-18 at 16 41 07" src="https://user-images.githubusercontent.com/35291291/82226115-61646d80-9926-11ea-96cf-0fe1a5fe4336.png">
